### DISCUSSION TITLE
How can we use grapesjs-alpha version?

### DISCUSSION THREAD

**Discussion by GautamSulok** [GrapesJS/grapesjs#6035] [Q&A]
How can we use grapesjs alpha version in our app, to achieve the following output without any customization, and just by using alpha version from grapes js
![image](https://github.com/user-attachments/assets/79479d8e-f0f7-462d-8a08-876cb446439c)

**Comment by artf** [comment]
I'd recommend taking a look a this [blog post](https://grapesjs.com/blog/meet-studio-sdk). It provides context about Studio and how to embed it, and just to clarify, Studio is an application, not a new version of GrapesJS framework.
